identity/src/commonMain/kotlin/com/android/identity/util/ByteArrayUtil.kt

```diff
@@ -0,0 +1,274 @@
+package com.android.identity.util
+
+import kotlinx.io.bytestring.ByteString
+import kotlinx.io.bytestring.buildByteString
+
+/**
+ * ByteArray extension functions for consistent reading and writing commonly used types.
+ */
+
+fun ByteArray.putInt8(offset: Int, value: Int, validRange: IntRange = Byte.MIN_VALUE..Byte.MAX_VALUE) {
```

Comment:
Where are the KDocs for all these functions? As a rule of thumb, all new API gets KDoc.